ListKit.xcodeproj/project.pbxproj

```diff
@@ -389,21 +389,24 @@
 			attributes = {
 				LastSwiftMigration = 0700;
 				LastSwiftUpdateCheck = 0700;
-				LastUpgradeCheck = 0700;
+				LastUpgradeCheck = 0800;
```

Comment:
I just upgraded my Xcode to `8.1` and rebuilt the project with that new version of Xcode. After the build, there are no changes in the git repository that I could commit though.

What exactly do I have to make sure the PR is coming from Xcode 8.1? I assume `LastUpgradeCheck` has to be set to `0810`?! Can this this be done manually or should it happen through an Xcode build? Are there other things to take care of?